src/universal-app/BUILD.bazel

```diff
@@ -59,21 +94,45 @@ devmode_esbuild(
     # Note: We add all linker-processed FW packages as dependencies here so that ESBuild will
     # map all framework packages to their linker-processed bundles from `tools/angular`.
     deps = LINKER_PROCESSED_FW_PACKAGES + [
-        ":server",
+        ":prerender_lib",
     ],
 )
 
-nodejs_test(
-    name = "server_test",
-    data = [
-        "index.html",
-        ":server_bundle",
-        ":theme_scss",
+npm_package_bin(
+    name = "prerender_test_bin",
+    outs = ["index-prerendered.html"],
+    args = ["$@"],
+    tool = ":prerender",
+)
+
+npm_package_bin(
+    name = "debug_prerender_bin",
+    outs = ["index.html"],
```

Comment:
I'd suggest calling this `index-prerendered-debug.html` then?

Reply:
Done.